### ISSUE TITLE
Standard Text Id (Key) Limited to 17 characters? 

### ISSUE THREAD

**Issue by msweier** [USACE/cwms-data-api#839]
Is it possible to expand the constraints on the standard text id? Seems like it's limited to 17 characters. Thanks

**Comment by MikeNeilson** [comment]
Yeah, but that constraint is handled within the schema so it'll have to wait for the next schema update after the one coming.

**Comment by msweier** [comment]
tagging @Enovotny for potential schema update to expand standard text id character limit. I would like to at least be able to use a prefix/suffix with a full timeseries id (plan on storing metadata for ts in there). 64 or 128 would probably be fine.
